examples/http-swagger/src/example/server.clj

```diff
@@ -142,6 +175,9 @@
       (swagger-ui/create-swagger-ui-handler
         {:path "/"
          :config {:validatorUrl nil
+         	        :urls [{:name "swagger", :url "swagger.json"} 
```

Comment:
Bad indent / tabs instead of spaces here.

Reply:
in #592